### PR TITLE
Ajit_Assignment_17_Done_(10 +25)mins

### PR DESCRIPTION
**Pull request by AjitSJune20**
image
image

### CODE REVIEW

File: src/ajitSJune20/CountCharOccuranceInString.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+public class CountCharOccuranceInString {
+	
+	void displayCount(String input) {
+		
+		for(int index=0;index<input.length();index++) {
+			char ch=input.charAt(index);
+			if(index == input.indexOf(ch))
+				if(input.indexOf(ch) != input.lastIndexOf(ch))
+					System.out.println(ch+" -> First Index : "+input.indexOf(ch) +" Last Index : "+input.lastIndexOf(ch));
```
Comment (KrishnaKTechnocredits) [inline src/ajitSJune20/CountCharOccuranceInString.java:13]: Perfect logic.

File: src/ajitSJune20/CountLetDigSpclChar.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+			else if(Character.isLowerCase(ch))
+				lower++;
+			else
+				special++;
+		}
+		/*System.out.println("Total Digit:--------> "+digit);
+		System.out.println("Total UpperCase:----> "+upper);
+		System.out.println("Total LowerCase:----> "+lower);
+		System.out.println("Total SpecialChar:--> "+special); */
+		System.out.println("Total Digit:--------> "+digit + "\nTotal UpperCase:----> "+upper+"\nTotal LowerCase:----> "+lower+"\nTotal SpecialChar:--> "+special);
```
Comment (KrishnaKTechnocredits) [inline src/ajitSJune20/CountLetDigSpclChar.java]: Ajit, here you forget to print total letters count.

Comment (AjitSJune20) [reply src/ajitSJune20/CountLetDigSpclChar.java]: Incorporated Changes.

Comment (AjitSJune20) [reply src/ajitSJune20/CountLetDigSpclChar.java]: image

File: src/ajitSJune20/CountLetDigSpclChar.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		
+		for(int index=0;index<str.length();index++) {
+			char ch=str.charAt(index);
+			if(Character.isDigit(ch))
+				digit++;
+			else if(Character.isUpperCase(ch))
+				upper++;
+			else if(Character.isLowerCase(ch))
+				lower++;
+			else if(Character.isLetter(ch))
```
Comment (KrishnaKTechnocredits) [inline src/ajitSJune20/CountLetDigSpclChar.java]: No you still not getting my point, I have said, without using letter variable and else if(Character.isLetter(ch)) block you can achieve the output. Still if you can't do it, will tell the solution.

Comment (AjitSJune20) [reply src/ajitSJune20/CountLetDigSpclChar.java]: Made Changes. I hope its correct this time. 
image

File: src/ajitSJune20/CountLetDigSpclChar.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+			else if(Character.isLowerCase(ch))
+				lower++;
+			else
+				special++;
+		}
+		/*System.out.println("Total Digit:--------> "+digit);
+		System.out.println("Total UpperCase:----> "+upper);
+		System.out.println("Total LowerCase:----> "+lower);
+		System.out.println("Total SpecialChar:--> "+special); */
+		System.out.println("Total Digit:--------> "+digit + "\nTotal Letters:------> "+(upper+lower)+ "\nTotal UpperCase:----> "+upper+"\nTotal LowerCase:----> "+lower+"\nTotal SpecialChar:--> "+special);
```
Comment (KrishnaKTechnocredits) [inline src/ajitSJune20/CountLetDigSpclChar.java:25]: Now it's perfect.

**Review by KrishnaKTechnocredits — APPROVED** [review]

**Pull request merged**